projects/addon-doc/src/components/documentation/documentation.template.html

```diff
@@ -16,6 +16,26 @@
             {{ texts[3] }}
         </th>
     </tr>
+    <tr class="t-row">
```

Comment:
I still think we should move it to a single point, not in the API. Not all components have text prompts so it wouldn't make sense in their API tab, and with text changed from `Language` to `Language of components` I think it works fine as a global select.

Reply:
fixed